docs(cloud): add load balancing article (#8036)
umbraco/UmbracoDocs: umbraco-cloud/optimize-and-maintain-your-site/optimize-performance/cache-configuration.md

---
description: >-

>   Enable a managed Redis cache on your Umbraco Cloud project. Compare the available SKUs, see pricing, and tune the cache via the CMS Hybrid Cache reference.

Reviewer: The article doesn't mention the pricing.

```suggestion
  Enable a managed Redis cache on your Umbraco Cloud project. Compare the available SKUs and tune the cache via the CMS Hybrid Cache reference.
```

{% hint style="info" %}
You enable and configure the managed Redis cache in the [Umbraco Cloud Portal](https://www.s1.umbraco.io/) under `Project -> Configuration -> Cache Configuration`.
{% endhint %}

## When to enable Cache

Enable Cache when you need:

* A distributed cache shared by all instances of your environment.
* A SignalR backplane to coordinate backoffice messages across instances.
* Lower database load on read-heavy sites.

{% hint style="info" %}

You do not need to enable Cache before turning on Load Balancing. If Cache is not already enabled on the environment, Umbraco Cloud provisions a managed Redis cache as part of enabling Load Balancing.

{% endhint %}

## How the managed Redis cache is used

When you enable Cache, Umbraco Cloud provisions a managed Redis instance for your environment. Redis serves two roles:

* **Distributed cache** — keeps cached state consistent across instances so every visitor sees the same content.
* **SignalR backplane** — coordinates real-time backoffice messages across all running instances.

> Umbraco CMS uses Microsoft's HybridCache under the hood. For details on how the CMS layers in-memory and distributed caching, and how to tune the cache, see the [HybridCacheOptions reference](https://docs.umbraco.com/umbraco-cms/reference/configuration/cache-settings#hybridcacheoptions).

Reviewer: Suggesting a small rewrite to avoid using "under the hood":

```suggestion
Umbraco CMS uses Microsoft's HybridCache for in-memory and distributed caching. See the [HybridCacheOptions reference](https://docs.umbraco.com/umbraco-cms/reference/configuration/cache-settings#hybridcacheoptions) for more details and for how to tune the cache.
```
